File: Python/Fibonacci.py
t1=0
t2=1
n=int(input("Enter the number of terms: "))
print("Fibonacci Series:")
print(t1)
print(t2)
for i in range(2, n):
  nextTerm = t1 + t2
  print(nextTerm)
  t1 = t2
  t2 = nextTerm

File: java/fibonacci.java
import java.util.*;
public class fibonacci {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number to enter: ");
        int n = sc.nextInt();
        int c=0;
        int a =0,b=1;
        System.out.print(a+" , "+b);
        for(int i=2;i<n;i++){
            c=a+b;
            a=b;
            b=c;
            System.out.print(" , " +c);
        }
    }
}
